packages/components/statistic/src/countdown.ts

```diff
@@ -1,29 +1,43 @@
-import { divide, floor, multiply, padStart, reduce } from 'lodash-unified'
-import dayjs from 'dayjs'
+import { padStart } from 'lodash-unified'
 import { buildProps, definePropType } from '@element-plus/utils'
 import type { StyleValue } from 'vue'
 import type { Dayjs } from 'dayjs'
 import type Countdown from './countdown.vue'
 export const countdownProps = buildProps({
+  /**
+   * @description Formatting the countdown display
+   */
   format: {
     type: String,
     default: 'HH:mm:ss',
   },
+  /**
+   * @description Sets the prefix of a countdown
+   */
   prefix: {
     type: String,
-    default: '',
```

Comment:
**Suggestion**
```ts
prefix: {
  type: String,
}
```
to
```ts
prefix: String
```

Same in other places.

Reply:
ok